torchao/experimental/kernels/cpu/linear/examples/torch_custom_op/run_custom_op.py

```diff
@@ -4,16 +4,21 @@
 # This source code is licensed under the license found in the
 # LICENSE file in the root directory of this source tree.
 
-from torch_custom_op import quantize, replace_linear_with_quantized_linear
-import torch
 import copy
 
-group_size = 16
+import torch
+from torch_custom_op import (
+    linear_a8sz_w_lowbit_reference_impl,
+    replace_linear_with_quantized_linear,
+)
+
+group_size = 256
 m = 1
 n = 4096
 k = 4096
-nbit = 4
-n_layers = 10
+nbit = 5
```

Comment:
wondering is this uint5 or int5?

Reply:
The 5bit kernel has quantized range [-32, 31].